packages/babel-plugin-transform-shorthand-properties/src/index.js

```diff
@@ -20,13 +20,22 @@ export default declare(api => {
           );
           func.returnType = node.returnType;
 
-          path.replaceWith(t.objectProperty(node.key, func, node.computed));
+          const computedKey = t.toComputedKey(node);
+          if (t.isStringLiteral(computedKey, { value: "__proto__" })) {
+            path.replaceWith(t.objectProperty(computedKey, func, true));
```

Comment:
```suggestion
          if (t.isStringLiteral(computedKey, { value: "__proto__" })) {
            const computedKey = t.toComputedKey(node);
            path.replaceWith(t.objectProperty(computedKey, func, true));
```
nit

Reply:
That will cause a `ReferenceError` because `computedKey` won’t be in scope for the `if` statement.